docs/document/docs/components/mobile/mobile-text.md

```diff
@@ -8,10 +8,10 @@ sidebar_label: MobileText
 
 The MobileText component allows the user to display a single line text element.
 
-<div class="sample-container" id="mobile-text">
-  <div id="sample-container__components" class="mobile"></div>
-</div>
-<script src="/js/samples/mobile/mobile-text.js"></script>
+import { MobileTextComponent } from "@site/static/js/samples/mobile/text.js"
+
+<MobileTextComponent />
```

Comment:
The default width setting seems wrong.
Please check the previous one.

Reply:
i have fixed